tests/test_profile.py

```diff
@@ -235,3 +235,17 @@ def test_that_links_in_the_services_page_return_200_code(self, base_url, seleniu
                 bad_urls.append(check_result)
 
         assert 0 == len(bad_urls), u'%s bad links found. ' % len(bad_urls) + ', '.join(bad_urls)
+
+    @pytest.mark.credentials
+    @pytest.mark.nondestructive
+    def test_new_user_cannot_see_groups_or_functional_areas(self, base_url, selenium, new_registered_user):
+        home_page = Home(base_url, selenium)
+        home_page.login(new_registered_user['email'], new_registered_user['password'])
+
+        assert not home_page.header.is_groups_menu_item_present
+        assert not home_page.is_groups_link_present
+        assert not home_page.is_functional_areas_link_present
```

Comment:
I left present for the menu item because it's hidden unless you click it and simplifies the test but I can mess with clicking, checking, clicking away if that's better. Changed the others to use visible.